code/modules/europa/psychics/faculties/creativity.dm

```diff
@@ -33,32 +34,57 @@
 		return TRUE
 	return FALSE
 
-/datum/psychic_power/electrocute
+/datum/psychic_power/spark/disrupt
+	name = "Disrupt"
+	description = "Scramble their electromagnetic field."
+	melee_power_cost = 10
+	time_cost = 100
+
+/datum/psychic_power/spark/disrupt/do_proximity(var/mob/living/user, var/mob/living/target)
+	if(..())
+		user.visible_message("<span class='danger'>\The [user]'s hands vomit a gout of crackling static and arcing lightning over \the [target]!</span>")
+		target.emp_act(pick(list(1,2,2,2,2,2,3,3,3,3,3,3,3,3)))
```

Comment:
You can weight a pick if you don't use a list. Right now it's a mess.
Syntax is either `pick( 1;1, 5;2, 8;3 )` or `pick( prob(1);1, prob(5);2, prob(8);3 )`